emsdk

```diff
@@ -251,7 +251,7 @@ def win_set_environment_variable_direct(key, value, system=True):
     win32api.RegSetValueEx(folder, key, 0, win32con.REG_EXPAND_SZ, value)
     if VERBOSE: print('Set key=' + key + ' with value ' + value + ' in registry.')
   except Exception as e:
-    if e[0] == 5 or e[2] == 'Access is denied.':
+    if 'Access is denied.' in str(e):
```

Comment:
That won't work on non-English Windows systems (e.g. Russian):

```
Failed to write environment variable PATH:
(5, 'RegOpenKeyEx', 'Отказано в доступе.')
```

Similarly below. I still think using `.args[0]` is a better fix.

Reply:
Ah right, thanks - the old code was doing that so I just assumed it was ok, but of course it isn't! :)

Ok, I changed it all to just use the code, and just using `e.args`.